### DISCUSSION TITLE
Need help! Where is the cdn link? Where are these verses stored, how can I access them? [See picture below]

### DISCUSSION THREAD

**Discussion by basilysf1709** [quran/discussions#11] [Q&A]
- "url": "AbdulBaset/Murattal/mp3/001004.mp3"
- I need to see where these verses are stored
![image](https://github.com/quran/discussions/assets/92009321/0c73a3e8-c842-4dd3-8f2d-fc15cf7a9eff)

**Comment by musalih** [comment]
Hi @basilysf1709. The base url is `https://verses.quran.com`
